elements/pfe-avatar/README.md

```diff
@@ -0,0 +1,69 @@
+PFElements Avatar Element
+
+`pfe-avatar` is an element for displaying a user's avatar image. If the user in question has provided a custom avatar, provide it and it will be displayed. If they don't, a nice colored pattern will be generated based on their name. A specific name will always generate the same pattern, so users' avatars will stay static without the need for storing their generated image.
+
+## Usage
+
+```html
+<pfe-avatar name="Eleanore Stillwagon"></pfe-avatar>
+
+<pfe-avatar name="Libbie Koscinski" shape="rounded" pattern="squares"></pfe-avatar>
+
+<pfe-avatar name="Blanca Rohloff" show-initials pattern="triangles"></pfe-avatar>
+
+<pfe-avatar name="Edwardo Lindsey" src="https://clayto.com/2014/03/rgb-webgl-color-cube/colorcube.jpg"></pfe-avatar>
+```
+
+## Attributes
+
+### name (observed) (required)
+
```

Comment:
@mwcz Should we remove the last sentence from the doc about the name being used to generate displayed initials?

Reply:
Yes! :+1: